src/style.css

```diff
@@ -1 +1,87 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
```

Comment:
Remove margin-right from the last stars in a row (use :last-child rule)
![image](https://github.com/mate-academy/layout_stars/assets/80700247/ce5909ab-b0cb-4527-ba10-737f73a05f6c)